.circleci/config.yml

```diff
@@ -211,7 +211,7 @@ jobs:
   test-creator-node:
     docker:
       # specify the version you desire here
-      - image: circleci/node:10.4
+      - image: circleci/node:10.16.3
```

Comment:
@vicky-g it may make sense to try upgrading all node versions in the circle config.yaml to this version of node and see if tests all still pass